File: 0789 - Fundamentos de linguagem JAVA/Semana7/ClassesV2/Aluno.py
from Turma import Turma


# al = Aluno("", "", "") <-  construtor(em py __init__) <- metodo responsavel por criar a class
class Aluno:
    nome: str
    idade: int
    turma: Turma

    def __init__(self, nome: str, idade: int, turma: Turma):
        self.nome = nome
        self.idade = idade
        self.turma = turma

    def __str__(self):
        return f"{self.nome}\t{self.idade}\t{self.turma}"

File: 5424 - Acesso móvel a sistemas de informação/heranca_v1/heranca_v1/Aluno.swift
//
//  Aluno.swift
//  heranca_v1
//
//  Created by Gonçalo Feliciano on 25/06/2024.
//

import Foundation


class Aluno: Pessoa{
    
    
    var turma: String
    
    
    init(nome: String, idade: Int, email: String, turma: String) {
    
        
        self.turma = turma
        super.init(nome: nome, idade: idade, email: email)

    }
    
    func funcDoAluno(){}
    
    func funcEmComum(){}
    
    override func getInfo() -> String {
        "nome: \(nome)\n\(idade)\n\(email)\n\(turma)"
    }
    
    
}
